src/apps/notifications/__test__/components/NotificationDetails.test.tsx

```diff
@@ -0,0 +1,143 @@
+  beforeEach(() => {
+    const queries = renderUI({
+      notification: notificationItem
+    });
+    queryByTestId = queries.queryByTestId;
+  });
+
+  it("should have the correct table headers", () => {
+    const enhancedTableHead = queryByTestId('enhancedTableHead');
+    const enhancedTableHeadData = enhancedTableHead ? enhancedTableHead.querySelectorAll("th"): [];
+
+    expect(enhancedTableHeadData.length).toEqual(4);
+    expect(enhancedTableHeadData[0].textContent).toBe("Concept ID");
+    expect(enhancedTableHeadData[1].textContent).toBe("Concept Type");
+    expect(enhancedTableHeadData[2].textContent).toBe("Statussorted descending");
+    expect(enhancedTableHeadData[3].textContent).toBe("Reasons");
+  });
+
+  it("should have the correct table data in appropriate columns - Parent Concept Imported", () => {
+    const tableRow = queryByTestId('testConceptID1');
+    const tableRowData = tableRow ? tableRow.querySelectorAll("td"): [];
+
+    expect(tableRowData.length).toEqual(4);
+    expect(tableRowData[0].textContent).toBe("testConceptID1");
+    expect(tableRowData[1].textContent).toBe("Parent");
+    expect(tableRowData[2].textContent).toBe("Imported");
+    expect(tableRowData[3].textContent).toBe("concept imported successfully");
+  });
+
+  it("should have the correct table data in appropriate columns - Parent Concept Failed", () => {
+    const tableRow = queryByTestId('testConceptID2');
+    const tableRowData = tableRow ? tableRow.querySelectorAll("td"): [];
+
+    expect(tableRowData.length).toEqual(4);
+    expect(tableRowData[0].textContent).toBe("testConceptID2");
+    expect(tableRowData[1].textContent).toBe("Parent");
+    expect(tableRowData[2].textContent).toBe("Skipped");
+    expect(tableRowData[3].textContent).toBe("concept import failed due to conflicts");
+  });
+
+  it("should have the correct table data in appropriate columns - Dependent Concept Imported", () => {
+    const tableRow = queryByTestId('testDependentConceptID1');
+    const tableRowData = tableRow ? tableRow.querySelectorAll("td"): [];
+
+    expect(tableRowData.length).toEqual(4);
+    expect(tableRowData[0].textContent).toBe("testDependentConceptID1");
+    expect(tableRowData[1].textContent).toBe("Dependent");
+    expect(tableRowData[2].textContent).toBe("Imported");
+    expect(tableRowData[3].textContent).toBe("concept imported successfully");
+  });
+
+  it("should have the correct table data in appropriate columns - Dependent Concept Failed", () => {
+    const tableRow = queryByTestId('testDependentConceptID2');
+    const tableRowData = tableRow ? tableRow.querySelectorAll("td"): [];
+
+    expect(tableRowData.length).toEqual(4);
+    expect(tableRowData[0].textContent).toBe("testDependentConceptID2");
+    expect(tableRowData[1].textContent).toBe("Dependent");
+    expect(tableRowData[2].textContent).toBe("Skipped");
+    expect(tableRowData[3].textContent).toBe("concept import failed due to conflicts");
```

Comment:
@tarunkumar-tw  can we have only one expect per test. Try to club the the expects into one if thats possible

Reply:
@rakesh-1010 Done. Fixed the broken unit tests. It is possible to combine all expects into one expect by check tableRow.textContent but it will not give us the clear picture of what column data went wrong etc if we have a single expect. 

This unit test is to validate a table row data and it will be clear if we have different expects for each column data in a row.

Reply:
@rakesh-1010 Updated unit tests to have only one expect per test.